src/app/personside/infotabs/oversikt/YtelserOversikt.tsx

```diff
@@ -82,12 +88,15 @@ function ForeldrepengerKomponent(props: { foreldrepenger: Foreldrepengerettighet
             valgt={false}
             ariaDescription="Vis foreldrepenger"
         >
-            <Normaltekst>ID dato: {props.foreldrepenger.rettighetFom}</Normaltekst>
+            <Normaltekst>
+                ID dato: {props.foreldrepenger.rettighetFom ? formaterDato(props.foreldrepenger.rettighetFom) : ''}
```

Comment:
Kan bruke funksjon for å hente ut id-dato her, så sikrer vi at det blir likt på tvers av applikasjonen

Reply:
`getForeldrepengerIdDato(foreldrepenger)`, ligger sammen med modellen